dspace-spring-rest/src/test/java/org/dspace/app/rest/GroupRestRepositoryIT.java

```diff
@@ -29,6 +36,23 @@
 
 public class GroupRestRepositoryIT extends AbstractControllerIntegrationTest {
 
+    @Test
+    public void createTest()
+        throws Exception {
+
+        ObjectMapper mapper = new ObjectMapper();
+        GroupRest groupRest = new GroupRest();
+        String groupName = "testGroup1";
+
+        groupRest.setName(groupName);
+
+        String authToken = getAuthToken(admin.getEmail(), password);
+        getClient(authToken).perform(post("/api/eperson/groups")
+                .content(mapper.writeValueAsBytes(groupRest)).contentType(contentType))
+                .andExpect(status().isCreated())
+                .andExpect(jsonPath("name", Matchers.containsString(groupName)));
+    }
```

Comment:
try to retrieve the created group via the GET call (this assure that the new object has been persisted)

Reply:
This has been fixed in the main `createTest()` method already. https://github.com/DSpace/DSpace/pull/2232/files#diff-5803046521b19bd520d333b334b9eb5eR55

Marking as resolved